tests/integration/it_utils.py

```
@@ -545,3 +548,48 @@ def compare_amm_values(val, val2, round_buffer):
             f"with round_buffer {round_buffer}"
         )
     return True
+
+
+def create_mpt_token_and_authorize_source(
+    issuer: Wallet,
+    source: Wallet,
+    client: SyncClient = JSON_RPC_CLIENT,
+    flags: List[int] = [],
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)